src/workloads/execution/DensifyTimeseriesCollection.yml

```diff
@@ -0,0 +1,451 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of $densify stage with a numeric step
+  in timeseries collections.
+
+GlobalDefaults:
+  dbname: &db test
+  batchSize: &batchSize 30000
+  fieldName: &field "numeric"
+  unitName: &unit null
+  index: &index
+    keys: {numeric: 1}
+    options: {name: "numeric"}
+  smallStepSize: &smallStep {^RandomInt: {min: 3, max: 3}}
+  largeStepSize: &largeStep {^RandomInt: {min: 8, max: 8}}
+  explicitBounds: &bounds [{^RandomInt: {min: 5000, max: 15000}}, {^RandomInt: {min: 20000, max: 40000}}]
```

Comment:
The max value in the documents is 20k no? So we should lower the total bounds to cover just the range that the documents actually cover + a bit, but this seems excessive. Don't think we should make the max bounds always over the existing documents

Reply:
Interesting, I got these bounds from DensifyNumeric. But I'll push changes like you described, bc you what you said does make more sense.